src/robot_interface/models/telemetry/entity.py

```diff
@@ -0,0 +1,14 @@
+import json
+
+
+class Entity:
+    """This is the base class for the model classes. It provides json serialization
+    methods."""
+
+    @classmethod
+    def from_json(cls, data):
+        jsonObj = json.loads(data)
+        return cls(**jsonObj)
+
+    def to_json(self):
+        return json.dumps(self, default=lambda o: o.__dict__, sort_keys=True, indent=2)
```

Comment:
Where are these classes from? Weren't they supposed to be generated and imported from Flotilla?

Reply:
Yes, they are here just to prove the concept. They will be imported from flotilla-openapi.